src/Perl6/Pod.nqp

```diff
@@ -327,6 +327,39 @@ class Perl6::Pod {
         return $*W.add_constant($type, 'type_new', |@pos, |%named);
     }
 
+    sub string2twine($S) {
+        # takes a simple string with unhandled formatting code
+        # and converts it into a twine data structure. primarily
+        # designed for a table cell.
+        # note all Z<> comments have already been removed.
+	my $s   := $S; # the raw string to be converted to a twine
+	my $ret := ''; # the cleaned string (i.e., without comments)
+	my $idx := nqp::index($s, '<'); # find the possible beginning of a formatting code
+        if $idx < 0 {
+            # no '<' found so return the simple string
+            return $s;
+        }
+
+	while $idx > -1 {
+            my $idx2 := nqp::index($s, '>', $idx+2); # and the end
+            nqp::die("FATAL:  Non-existent closing '>' for inline pod comment in string '$s' in Table $table_num") if $idx2 < 0;
```

Comment:
Maybe: "Couldn't find terminator '>' for inline pod comment..." so it matches closer to the wording we use for other error messages.

Reply:
ok, will change...